tests/neo.UnitTests/Network/P2P/UT_LocalNode.cs

```diff
@@ -20,15 +20,17 @@ public void Init()
         [TestMethod]
         public void TestDefaults()
         {
-            Assert.AreEqual(0, LocalNode.Singleton.ListenerTcpPort);
-            Assert.AreEqual(0, LocalNode.Singleton.ListenerWsPort);
-            Assert.AreEqual(3, LocalNode.Singleton.MaxConnectionsPerAddress);
-            Assert.AreEqual(10, LocalNode.Singleton.MinDesiredConnections);
-            Assert.AreEqual(40, LocalNode.Singleton.MaxConnections);
-            Assert.AreEqual(0, LocalNode.Singleton.UnconnectedCount);
+            var localNode = new LocalNode(testBlockchain);
```

Comment:
I suppose this will cause some runtime error?

Reply:
Yes I will fix it

Reply:
We can wait https://github.com/neo-project/neo/pull/2302#discussion_r571594653 or remove this UT